### ISSUE TITLE
Panic on start

### ISSUE THREAD

**Issue by ghost** [Adamkob12/minecraft_bevy#1]
`cargo run` panics:

```
╰─$ cargo run                                                                                                                                                                                        101 ↵
    Finished dev [unoptimized + debuginfo] target(s) in 0.13s
     Running `target/debug/minecraft_bevy`
2023-10-19T15:23:26.526037Z  INFO bevy_winit::system: Creating new window "App" (0v0)
2023-10-19T15:23:26.526291Z  INFO winit::platform_impl::platform::x11::window: Guessed window scale factor: 1    
2023-10-19T15:23:27.005880Z  INFO bevy_render::renderer: AdapterInfo { name: "AMD Radeon RX 6600 (RADV NAVI23)", vendor: 4098, device: 29695, device_type: DiscreteGpu, driver: "radv", driver_info: "Mesa 23.1.8", backend: Vulkan }
2023-10-19T15:23:27.127652Z  INFO bevy_diagnostic::system_information_diagnostics_plugin::internal: SystemInfo { os: "Linux 38 Fedora Linux", kernel: "6.5.7-200.fc38.x86_64", cpu: "Intel(R) Core(TM) i5-8600 CPU @ 3.10GHz", core_count: "6", memory: "31.3 GiB" }
2023-10-19T15:23:27.342184Z  INFO minecraft_bevy: 
Internal Log:
Meshes have been loaded
2023-10-19T15:23:27.858739Z  INFO minecraft_bevy: 
Internal Log:
Chunk entities have been successfully spawned
thread 'Compute Task Pool (0)' panicked at src/chunk/systems.rs:136:34:
attempt to subtract with overflow
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
Encountered a panic in system `minecraft_bevy::chunk::systems::cull_sides_of_mesh`!
Encountered a panic in system `bevy_app::main_schedule::Main::run_main`!
```

**System:** Fedora Workstation 38 (Wayland), rust 1.73.0

**Comment by Adamkob12** [comment]
run with the release flag ( cargo run --release) , for now, will fix soon

**Comment by duaneking** [comment]
This panic is fixed by #3
